schemas/groups/electrical_ac.json

```diff
@@ -0,0 +1,998 @@
+{
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-03/schema",
+    "id": "https://signalk.github.io/specification/schemas/groups/electrical_ac.json#",
+    "description": "Schema describing electric measurements pertaining to shore power, AC generators, etc.",
+    "title": "AC Electrical Properties",
+    "properties": {
+        "bus1": {
+            "type": "object",
+            "title": "Bus #1",
+            "description": "Main AC power bus on the vessel",
+            "properties": {
+                "phaseA": {
+                    "type": "object",
+                    "title": "Bus #1, Phase A",
+                    "properties": {
+                        "timestamp": {
+                            "$ref": "../definitions.json#/definitions/timestamp"
+                        },
+                        "source": {
+                            "$ref": "../definitions.json#/definitions/source"
+                        },
+                        "lineLineVoltage": {
```

Comment:
@timmathews maybe the `To` would be good to include, I agree with @tkurki that `lineLine` sounds weird.